lib/includes/utility.h

```diff
@@ -1,10 +1,8 @@
 #pragma once
 
-#include <iostream>
```

Comment:
I've removed these, since they are not needed **for this header**. However, new I've forced to add these includes to a number of **cpp** modules, because they was relying on this header. The reason of this - do not clutter the user space with potentially unneeded includes. If some module requires an header, it should include it explicitly.